modules/create-storefront-x/template/magento/modules/demo-magento/sw/manifest.js

```diff
@@ -0,0 +1,33 @@
+export default {
+  start_url: '/',
```

Comment:
I can imagine being a bit more creative here. Manifest could be its own concept where every field in the manifest object would be represented by a file and the Manifest concept would pick all of those files and merge them into one object. This way you could easy extend/override each of these fields instead of copying the whole object.

`my-module/serviceWorker/manifest/start_url.ts`

```
export default '/'
```